src/scripts/main.js

```diff
@@ -1 +1,78 @@
 'use strict';
+
+function notification(value) {
+  const div = document.createElement('div');
+
+  div.setAttribute('data-qa', 'notification');
+  div.classList = value[0];
+  div.innerText = value[1];
+  document.body.appendChild(div);
+}
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', (e) => {
+    const value = ['success', 'First promise was resolved'];
+
+    resolve(value);
+
+    e.preventDefault();
```

Comment:
The event listener for the 'click' event resolves the promise immediately upon any click, which might not be the intended behavior if the promise should only resolve under specific conditions. Consider verifying the event target or other conditions before resolving.